test/unittest/rtps/persistence/check_guid.py

```diff
@@ -0,0 +1,21 @@
+import sqlite3
```

Comment:
Missing license

Reply:
Missing docstring in public module (flake8 D100).
Since the module takes arguments, I'd rather have a small docstring defining the interface, how to use it, etc.